frontend/src/lib/components/LemonButton/index.tsx

```diff
@@ -15,15 +19,42 @@ export function LemonButton({
     className,
     ...buttonProps
 }: LemonButtonProps): JSX.Element {
-    return (
+    const { onClick, ...buttonPropsInternal } = buttonProps
```

Comment:
Why do we expand the extra properties twice? Worth just passing `buttonProps` down